src/services/Blockchain/useEvaluateTransaction.ts

```diff
@@ -24,7 +24,7 @@ export async function evaluateTransaction(
   try {
     const methods = await contract?.callStatic
     const balance = await methods?.[methodName](...args)
```

Comment:
Wrong naming this is not a balance. 
` const balance = await methods?.[methodName](...args) ` 
it should be
` const response = await methods?.[methodName](...args) ` 

I changed it before. I don't know how this change reverted. Whatever this is, a blockchain service you can't change. It is not a balance